__tests__/app.test.js

```diff
@@ -65,23 +65,78 @@ describe("app", () => {
                 expect(typeof res.body.topics).toBe("object")
                 expect(res.body.topics).toHaveLength(3)
             })
+        });
+    })
+
+    describe.only("GET /api/articles/:article_id/comments", () => {
+        test("should return an array of objects with expected properties", () => {
+            return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+                const result = res.body.comments
 
+                for (var key in result) {
+                    var obj = result[key];
```

Comment:
No use of var to declare variables please. These should be let, but tbh this shouldn't be a for in either and therefore these vars will go